nautobot/core/testing/api.py

```diff
@@ -231,9 +232,9 @@ def get_filterset(self):
         def get_depth_fields(self):
             """Get a list of model fields that could be tested with the ?depth query parameter"""
             depth_fields = []
-            for field in self.model._meta.fields:
+            for field in self.model._meta.get_fields():
                 if not field.name.startswith("_"):
-                    if isinstance(field, (ForeignKey, GenericForeignKey, ManyToManyField, core_fields.TagsField)) and (
+                    if isinstance(field, (ForeignKey, ManyToManyField, core_fields.TagsField)) and (
```

Comment:
I had some test failures around GFK fields when I fixed the logic so that we were actually correctly detecting depth fields. I'll revisit and see whether it should be reintroduced.